sdk/translation/azure-ai-translation-document/tests/test_model_updates.py

```diff
@@ -0,0 +1,253 @@
+        doc_input_positional = DocumentTranslationInput(
+                source_url=source_container_url,
+                targets=[TranslationTarget(target_url=target_container_url, language="fr")]
+        )
+        assert doc_input_positional is not None
+        assert doc_input_positional.source_url is not None
+        assert doc_input_positional.targets and doc_input_positional.targets[0].target_url and doc_input_positional.targets[0].language is not None
+
+        # Using keyword-only arguments to specify additional optional parameters
+        doc_input_keyword = DocumentTranslationInput(
+            source_container_url,
+            [TranslationTarget(target_url=target_container_url, language="fr")],
+            source_language="en",
+            storage_type="FOLDER",
+            storage_source="AzureBlob",
+            prefix="start_",
+            suffix="_end"
+        )
+        self.validate_document_translation(doc_input_keyword)
+
+        # Creating an instance using a dictionary to pass parameters
+        params = {
+            "source_url": source_container_url,
+            "targets": [TranslationTarget(target_url=target_container_url, language="fr")],
+            "source_language": "en",
+            "storage_type": "FOLDER",
+            "storage_source": "AzureBlob",
+            "prefix": "start_",
+            "suffix": "_end"
+        }
+        doc_input_dict = DocumentTranslationInput(**params)
+        self.validate_document_translation(doc_input_dict)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_translation_target_args(self, **kwargs): 
+        # Using keyword arguments to specify additional optional parameters
+        target_keyword = TranslationTarget(
+            target_url="https://t7d8641d8f25ec940prim.blob.core.windows.net/target-67890",
+            language="es",
+            category_id="general",
+            glossaries=[TranslationGlossary(glossary_url="https://glossaryfile.txt", file_format="txt")],
+            storage_source="AzureBlob"
+        )
+        self.validate_translation_target(target_keyword)
+
+        # Creating an instance using a dictionary to pass parameters
+        params = {
+            "target_url": "https://t7d8641d8f25ec940prim.blob.core.windows.net/target-67890",
+            "language": "es",
+            "category_id": "general",
+            "glossaries": [TranslationGlossary(glossary_url="https://glossaryfile.txt", file_format="txt")],
+            "storage_source": "AzureBlob"
+        }
+        target_dict = TranslationTarget(**params)
+        self.validate_translation_target(target_dict)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_translation_glossary_args(self, **kwargs):
+        # Using keyword arguments to specify additional optional parameters
+        glossary_keyword = TranslationGlossary(
```

Comment:
Same with this model, missing required positional params test

Reply:
Added